lib/jellyfish/config_reader.ex

```diff
@@ -246,6 +246,10 @@ defmodule Fishjam.ConfigReader do
     get_env("FJ_GIT_COMMIT", "dev")
   end
 
+  def read_server_api_token() do
+    get_env("FJ_SERVER_API_TOKEN", nil)
```

Comment:
Nitpick: `nil` is the default
```suggestion
    get_env("FJ_SERVER_API_TOKEN")
```